README.md

```diff
@@ -163,16 +164,16 @@ relics[0].describe()
 
 
 ### Config<a name="config"></a>
-A json text file named config located in ~/reliquery
+A json text file named config needs to be created in ~/reliquery 
 <br />
-Default looks like...
+It can be setup to look like this...
 ```json
 {
   "default": {
     "storage": {
       "type": "File",
       "args": {
-        "root": "/home/user/reliquery"
+        "root": "~/reliquery"
```

Comment:
Im not certain on this, but I don't believe "~" works for windows paths. I took Brett's feedback mean more of placeholder like the one you did for the Dropbox access token. 
`"root": "<ENTER_PATH_TO_STORE_YOUR_RELIQUERY"`
 
Another thought is to maybe add a note about the differing paths for linux/macos and winndows
`"~/reliquery" vs. "C:/User/<username>/reliquery"`